### ISSUE TITLE
about "loss" when training the model

### ISSUE THREAD

**Issue by FantDing** [eragonruan/text-detection-ctpn#94]
Hi! When I train the model, because of the sufficient graphic memory, I change the model like this
```python
(self.feed('data')
         .conv(3, 3, 64, 1, 1, name='conv1_1')
         .max_pool(2, 2, 2, 2, padding='VALID', name="conv1_1")
         .conv(3, 3, 64, 1, 1, name='conv1_2')
         .max_pool(2, 2, 2, 2, padding='VALID', name='pool1')
         .conv(3, 3, 128, 1, 1, name='conv2_1')
         .conv(3, 3, 128, 1, 1, name='conv2_2')
         .max_pool(2, 2, 2, 2, padding='VALID', name='pool2')
         .conv(3, 3, 256, 1, 1, name='conv3_1')
         # .conv(3, 3, 256, 1, 1, name='conv3_2')
         # .conv(3, 3, 256, 1, 1, name='conv3_3')
         .max_pool(2, 2, 2, 2, padding='VALID', name='pool3')
         .conv(3, 3, 512, 1, 1, name='conv4_1')
         # .conv(3, 3, 512, 1, 1, name='conv4_2')
         # .conv(3, 3, 512, 1, 1, name='conv4_3')
         .max_pool(2, 2, 2, 2, padding='VALID', name='pool4')
         .conv(3, 3, 512, 1, 1, name='conv5_1')
         .conv(3, 3, 512, 1, 1, name='conv5_2')
         .conv(3, 3, 512, 1, 1, name='conv5_3'))
```
However, the loss shaked violently

![image](https://user-images.githubusercontent.com/21337200/35765681-1fb83506-0904-11e8-9968-e194caffeff2.png)

![image](https://user-images.githubusercontent.com/21337200/35765670-dea3530c-0903-11e8-8ff0-4772a06f826f.png)

how to solve that? Thank you

**Comment by cipri-tom** [comment]
Maybe try decreasing the learning rate ? Or using a different optimiser

**Comment by FantDing** [comment]
@cipri-tom Thanks, I will have a try

**Comment by FantDing** [comment]
@cipri-tom What's the final value of `total loss`  approximately

**Comment by cipri-tom** [comment]
Mine goes down 0.08 in the final iterations.

But since I posted the comment I also trained on some dataset where the loss had spikes like yours. I couldn't identify the problem, but the training completed fine

**Comment by eragonruan** [comment]
@FantDing recheck the data, there may be some annotation error

**Comment by FantDing** [comment]
@eragonruan Thank you! The high loss may caused by the model, which I remove lots of layers from. Now, I fixed VGG16 layers' parameter. It takes about 0.5s per iter to train, which is more faster than before. However, the `total loss` is about `0.2` which is not very ideal.
